docs/sources/mimir/release-notes/v2.15.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Grafana Mimir version 2.15 release notes"
+menuTitle: "V2.15 release notes"
+description: "Release notes for Grafana Mimir version 2.15"
+weight: 1100
+---
+
+# Grafana Mimir version 2.15 release notes
+
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.15 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+S2 compression for gRPC is now supported using the following flags:
+
+- `-alertmanager.alertmanager-client.grpc-compression=s2`
+- `-ingester.client.grpc-compression=s2`
+- `-querier.frontend-client.grpc-compression=s2`
+- `-querier.scheduler-client.grpc-compression=s2`
+- `-query-frontend.grpc-client-config.grpc-compression=s2`
+- `-query-scheduler.grpc-client-config.grpc-compression=s2`
+- `-ruler.client.grpc-compression=s2`
+- `-ruler.query-frontend.grpc-client-config.grpc-compression=s2`
+
+Distributors now support `lz4` OTLP compression, and can be deployed in multiple availability zones.
+
+The ruler's `<prometheus-http-prefix>/api/v1/rules` endpoint now supports `exclude_alerts`, `group_limit`, and `group_next_token` parameters.
+
+mimirtool's analyze ruler/prometheus commands now support bearer tokens.
+
+HTTP client settings can now be tuned for GCS and Azure backends via an `http` block or corresponding CLI flags.
+
+The compactor now refreshes deletion marks concurrently when updating the bucket index.
+
+The number of Memcached replicas can now be independently set for each type of cache.
+
+## Important changes
+
+In Grafana Mimir 2.15, the following behavior has changed:
+
+The following alertmanager metrics are not exported for a `user` when the metric value is zero:
+
+- `cortex_alertmanager_alerts_received_total`
+- `cortex_alertmanager_alerts_invalid_total`
+- `cortex_alertmanager_partial_state_merges_total`
+- `cortex_alertmanager_partial_state_merges_failed_total`
+- `cortex_alertmanager_state_replication_total`
+- `cortex_alertmanager_state_replication_failed_total`
+- `cortex_alertmanager_alerts`
+- `cortex_alertmanager_silences`
+
+PromQL compatibility has been upgraded from Prometheus 2.0 to 3.0. The following has changed as a result:
```

Comment:
Is there a Prometheus doc we can link to here that explains these in more detail?

Reply:
Maybe [this](https://prometheus.io/docs/prometheus/3.0/migration/#promql). The native histograms change isn't mentioned on that page, though, afaict.